<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboutClass</class>
 <widget class="QMainWindow" name="aboutClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Registro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/imgs/images/window_icon.ico</normaloff>:/imgs/images/window_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/res.qrc">:/imgs/images/about_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registro</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVersion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Version: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAuthor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyright: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLicense">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>License: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBuild">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Build date: </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
